wallet-new/src/Cardano/Wallet/Util.hs

```diff
@@ -0,0 +1,25 @@
+
+-- | Module for small utility functions.
+module Cardano.Wallet.Util
+       ( headToLower
+       , stripFieldPrefix
+       , mkJsonKey
+       ) where
+
+import           Universum
+
+import           Data.Char (isUpper, toLower)
+
+--
+-- String manipulation utils
+--
+
+headToLower :: String -> Maybe String
+headToLower []     = Nothing
+headToLower (x:xs) = Just $ toLower x : xs
+
+stripFieldPrefix :: String -> String
+stripFieldPrefix = dropWhile (not . isUpper)
+
+mkJsonKey :: String -> String
+mkJsonKey s = fromMaybe s . headToLower $ stripFieldPrefix s
```

Comment:
Note to myself: Maybe in the future we could kill these ad-hoc `Util` modules in favour of a Wallet Prelude, in the same vein of:

https://www.reddit.com/r/haskell/comments/43du5b/simon_meier_a_primer_to_commercial_haskell/